ingenialink/ethercat/servo.py

```diff
@@ -150,57 +160,102 @@ def __init__(
         slave: "CdefSlave",
         slave_id: int,
         dictionary_path: str,
+        connection_timeout: float,
```

Comment:
connection_timeout param gets servo_status_listener from network, instead sdo_read_timeout. In addition, sdo_read_timeout is in nano seconds, and connection_timeout should be in second

Reply:
You're right, thanks for noticing.